dockerfile

```
@@ -0,0 +1,16 @@
+FROM python:3.8-alpine
+
+WORKDIR /app
+
+COPY ./python/requirements.txt /app/requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY ./python/ .
+COPY ./db/ /app/db/
+
+RUN addgroup -S mercari && adduser -S trainee -G mercari
+USER trainee
```

imo: こうしないとpermission deniedになると思います
```suggestion
RUN addgroup -S mercari && adduser -S trainee -G mercari
RUN chown -R trainee:mercari /app
USER trainee
```